scripts/Muon/GUI/ElementalAnalysis2/plotting_widget/EA_plot_widget.py

```diff
@@ -0,0 +1,89 @@
+    EAPlotDataPanePresenter
+from Muon.GUI.Common.plot_widget.base_pane.base_pane_view import BasePaneView
+
+
+class EAPlotWidget(object):
+
+    def __init__(self, context=None, get_active_fit_results=lambda: [], parent=None):
+
+        self.data_model = EAPlotDataPaneModel(context)
+        # self.fit_model = PlotTimeFitPaneModel(context)
+
+        # The plotting canvas widgets
+        self.plotting_canvas_widgets = {self.data_model.name: PlottingCanvasWidget(parent, context=
+                                        context.plot_panes_context[self.data_model.name], plot_model=self.data_model)}
+
+        # self.plotting_canvas_widgets[self.fit_model.name] = PlottingCanvasWidget(parent, context=
+        # context.plot_panes_context[self.fit_model.name],plot_model=self.fit_model)
+
+        # The UI view
+        self._view1 = BasePaneView(parent)
+        self._view1.add_canvas_widget(self.plotting_canvas_widgets[self.data_model.name].widget)
+
+        # self._view2 = BasePaneView(parent)
+        # self._view2.add_canvas_widget(self.plotting_canvas_widgets[self.fit_model.name].widget)
+        # set up presenter
+        self.view = MainPlotWidgetView(parent)
+        self.model = EAPlotDataPaneModel(context)
+        # generate the presenter
+
+        self.data_mode = EAPlotDataPanePresenter(self._view1, self.data_model,
+                                                 context, self.plotting_canvas_widgets[self.data_model.name].presenter)
+        # self.fit_mode = PlotFitPanePresenter(self._view2, self.fit_model,
+        # context, self.plotting_canvas_widgets[self.fit_model.name].presenter)
+
+        self.presenter = MainPlotWidgetPresenter(self.view,
+                                                 [self.data_mode])  # , self.fit_mode])
+        self._current_plot_mode = self.presenter.get_plot_mode
+        self.presenter.set_plot_mode_changed_slot(self.handle_plot_mode_changed_by_user)
+
+        for observer in self.presenter.workspace_replaced_in_ads_observers:
+            context.update_plots_notifier.add_subscriber(observer)
+        for observer in self.presenter.workspace_deleted_from_ads_observers:
+            context.deleted_plots_notifier.add_subscriber(observer)
+
+    def set_plot_view(self, plot_mode):
+        self.view.set_plot_mode(plot_mode)
+
+    @property
+    def data_changed_observers(self):
+        return self.presenter.data_changed_observers
+
+    @property
+    def workspace_deleted_from_ads_observers(self):
+        return self.presenter.workspace_deleted_from_ads_observers
+
+    @property
+    def rebin_options_set_observers(self):
+        return self.presenter.rebin_options_set_observers
+
+    @property
+    def data_index(self):
+        return self.view.get_index(self.data_mode.name)
+
+    @property
+    def fit_index(self):
+        return self.view.get_index(self.fit_mode.name)
+
+    def close(self):
+        self.view.close()
+
+    def handle_plot_mode_changed_by_user(self):
```

Comment:
do you want this functionality?

Reply:
I was not sure what this did as it was from the  muon plot widget

Reply:
This keeps the quickedit synced between the data and fitting panes

Reply:
It is not needed for this pull request as the fitting tab hasnt been added yet

Reply:
can we delete it then?